examples/groups.dtd

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Internally this is groupsfeed.dtd. It looks like the name was changed elsewhere during development, and xmlgroups.dtd works for me, if we're willing to rename the internal file eventually.